app/src/main/java/com/poti/android/presentation/alarm/list/AlarmListViewModel.kt

```
@@ -0,0 +1,24 @@
+class AlarmListViewModel @Inject constructor() : BaseViewModel<AlarmListUiState, AlarmListUiIntent, AlarmListUiEffect>(
+    initialState = AlarmListUiState(),
+) {
+    override fun processIntent(intent: AlarmListUiIntent) {
+        when (intent) {
+            AlarmListUiIntent.OnBackClick -> sendEffect(AlarmListUiEffect.NavigateBack)
+            AlarmListUiIntent.OnSettingClick -> sendEffect(AlarmListUiEffect.NavigateToSetting)
+            is AlarmListUiIntent.OnAlarmClick -> handleAlarmClick(intent.alarm)
+        }
+    }
+
+    private fun handleAlarmClick(alarm: Notification) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)